client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -53,10 +53,10 @@ const siteVertical: Reducer<
 	return state;
 };
 
-const pageLayouts: Reducer<
-	string[],
-	ReturnType< typeof Actions[ 'togglePageLayout' ] >
-> = ( state = [], action ) => {
+const pageLayouts: Reducer< string[], ReturnType< typeof Actions[ 'togglePageLayout' ] > > = (
+	state = [],
+	action
+) => {
```

Comment:
This triggered a codeowner review, which I suspect was accidental. Care to roll that back and we can remove the review request?